site/home.md

```diff
@@ -33,4 +33,4 @@ Want to be added here? [Here's how.](https://github.com/VickiLanger/OpenGates#re
 ### Contributors of OpenGates
 OpenGates would be nothing without our awesome contributors. We appreciate single person who has take the time to help OpenGates reduce gatekeeping in our dev communities.
 
-[![image of contributors generated by https://contributors-img.web.app/ pulling from https://github.com/VickiLanger/OpenGates/graphs/contributors](https://contributors-img.web.app/image?repo=VickiLanger/OpenGates)](https://github.com/VickiLanger/OpenGates/graphs/contributors)
\ No newline at end of file
+[![image of contributors generated by https://contributors-img.web.app/ pulling from https://github.com/VickiLanger/OpenGates/graphs/contributors](https://contributors-img.web.app/image?repo=VickiLanger/OpenGates)](https://github.com/VickiLanger/OpenGates/graphs/contributors)
```

Comment:
This diff is just a newline addition. You can verify it by hiding whitespace changes under the gear ⚙️ setting toggle above

Reply:
Fantastic catch! Thanks for fixing this.